tests/test_search.py

```diff
@@ -33,16 +34,21 @@ def test_search_with_one_line(self, base_url, selenium):
     def test_search_with_multiple_lines(self, base_url, selenium):
         csp = CrashStatsHomePage(selenium, base_url).open()
         cs_super = csp.header.click_super_search()
-        cs_super.select_field('product')
-        cs_super.select_operator('has terms')
         cs_super.click_new_line()
-        cs_super.select_field('release channel')
-        cs_super.select_operator('has terms')
         # select the 2nd line
+        cs_super.select_field('release channel')
+        cs_super.select_operator('1', 'has terms')
         cs_super.select_match('1', 'nightly')
         cs_super.click_search()
 
         assert cs_super.are_search_results_found
```

Comment:
If we decide to add a comment to the earlier asserts (https://github.com/mozilla/Socorro-Tests/pull/382/files#r68177576), it'd be good to do so here, as well (again, I won't block my r+ on that).